addon/lib/document.js

```diff
@@ -127,6 +128,34 @@ export default Base.extend({
       console.debug(`${label}:`, any);
       return any;
     });
+    documentJexl.addTransform("min", (arr) => {
+      const nums = onlyNumbers(arr);
+      return nums.length ? Math.min(...nums) : null;
+    });
+    documentJexl.addTransform("max", (arr) => {
+      const nums = onlyNumbers(arr);
+      return nums.length ? Math.max(...nums) : null;
+    });
+    documentJexl.addTransform("round", (num, places) =>
+      isNaN(num)
+        ? null
+        : places
+        ? Math.round(num * Math.pow(10, places)) / Math.pow(10, places)
```

Comment:
Drop the line below and provide a default value of `0` for the argument `places`. `Math.pow(10, 0)` will result in `1` so it does the right thing. This way we can get rid of the nested ternary which is really hard to read